src/scenes/NftDetailPage/NavigationHandle.tsx

```diff
@@ -1,48 +1,63 @@
 import { useCallback, useMemo } from 'react';
 import styled from 'styled-components';
-import Gradient from 'components/core/Gradient/Gradient';
-import { BodyRegular } from 'components/core/Text/Text';
+import ActionText from 'components/core/ActionText/ActionText';
 import colors from 'components/core/colors';
 import transitions from 'components/core/transitions';
 import Spacer from 'components/core/Spacer/Spacer';
 import { Directions } from 'components/core/enums';
 import { useRouter } from 'next/router';
+import breakpoints, { pageGutter } from 'components/core/breakpoints';
+import { useIsMobileOrMobileLargeWindowWidth } from 'hooks/useWindowSize';
+import ArrowLeft from 'public/icons/arrow_left.svg';
+import ArrowRight from 'public/icons/arrow_right.svg';
 
 const ARROWS = new Map<number, string>([
   [Directions.LEFT, '←'],
   [Directions.RIGHT, '→'],
 ]);
 
+const MOBILE_ARROWS = new Map<number, any>([
+  [Directions.LEFT, <ArrowLeft />],
+  [Directions.RIGHT, <ArrowRight />],
+]);
+
 const HOVER_TEXT = new Map<number, string>([
   [Directions.LEFT, 'Prev'],
   [Directions.RIGHT, 'Next'],
 ]);
 
 type Props = {
   direction: Directions;
+  username: string;
+  collectionId: string;
   nftId: string;
 };
 
-function NavigationHandle({ direction, nftId }: Props) {
-  const arrow = useMemo(() => ARROWS.get(direction) ?? '', [direction]);
+function NavigationHandle({ direction, username, collectionId, nftId }: Props) {
+  const isMobileOrMobileLarge = useIsMobileOrMobileLargeWindowWidth();
+
+  const arrow = useMemo(
+    () => (isMobileOrMobileLarge ? MOBILE_ARROWS.get(direction) : ARROWS.get(direction) ?? ''),
+    [isMobileOrMobileLarge, direction]
+  );
```

Comment:
Might need review. I assume this is the best way to handle dynamic rendering (icons on mobileLarge and down, text on tablet and higher)

Reply:
looks good!